### PR TITLE
changed the navbar button from signup to login

### PR DESCRIPTION
**Pull request by Cheyenneb96**
## Description

changed the nav bar button so say login as requested

Fixes # (issue)

## Loom Video

https://www.loom.com/share/b136009157d04a4fbe3293b8e738c263

## Type of change

Please delete options that are not relevant.

- [ .] feature (non-breaking change )

## Checklist:

- [. ] My code follows the style guidelines of this project
- [. ] I have performed a self-review of my own code
- [. ] I have removed unnecessary comments/console logs from my code
- [. ] I have made corresponding changes to the documentation if necessary (optional)
- [ .] My changes generate no new warnings
- [. ] I have checked my code and corrected any misspellings
- [. ] No duplicate code left within changed files
- [. ] Size of pull request kept to a minimum
- [. ] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

**Review by mark-escosura — APPROVED** [review]
Nice! I was literally just thinking about doing this. Good thing I caught this PR right away! Approved!

**Review by katherineyevsukov — APPROVED** [review]
Looks good to go!

**Review by mcmillanj — APPROVED** [review]
good work on sign

**Review by seanmp3 — APPROVED** [review]
Important fix to make the user experience for returning users more natural. Also Waylon put me on to this, so I'll spread the love letting you know about this as well; if you put an x with no spaces around the brackets in the checklists they become little boxes that you can check on and off without editing the file.

**Review by manny-glez — APPROVED** [review]
Looks great, good change.

File: src/components/common/NavBar.js [hunk 1]
```diff
@@ -58,7 +58,7 @@ export default function MainNav() {
           <button>CONTACT US</button>
         </div>
         <div className="nav navSignup">
-          <button>SIGN UP</button>
+          <button>LOGIN</button>
```
Comment (paulstgermain) [inline src/components/common/NavBar.js:61]: We should also be sure to hook this up to route us to the Login page!

**Pull request merged**